src/test/java/edu/kis/vh/nursery/stacks/IntArrayStackTest.java

```diff
@@ -0,0 +1,86 @@
+package edu.kis.vh.nursery.stacks;
+
+import org.junit.Assert;
+import org.junit.Test;
+
+public class IntArrayStackTest {
+    @Test
+    public void testPop() {
+        IntArrayStack stack = new IntArrayStack();
+        final int EMPTY_STACK_HEAD_INDEX = -1;
```

Comment:
The original constant should be used (it is better to use a constant from stacks interface).